debezium-bom/pom.xml

```diff
@@ -54,6 +54,12 @@
         <!-- Azure Identity -->
         <version.azure.identity>1.15.4</version.azure.identity>
 
+        <!-- Azure Core -->
+        <version.azure.core>1.57.0</version.azure.core>
+
+        <!-- MSAL4J-->
+        <version.msal4j>1.23.1</version.msal4j>
```

Comment:
The `com.azure:azure-storage-blob` already brings in azure-core 1.54.0 and azure-identity 1.15.4 brings in msal4j 1.19.1. We should ideally try and make sure that we are aligning all Azure dependencies so that we don't run into potential conflicts across different component versions being incompatible.

Reply:
You’re right, I removed it from the pom.xml. Since the issue is about MS SQL Server, I added the Azure SDK BOM to the SQL Server connector as it includes all the dependencies needed for managed identity authentication to work. I’ll make sure to test it properly.